templates/personal-settings.php

```diff
@@ -23,7 +23,7 @@
         <div id="pubUrlLabel"><?php p($l->t('Copied')); ?></div>
 
         <h3>
-            <?php p($l->t('Embeddable URL')); ?>
+            <?php p($l->t('Embeddable URL:')); ?>
```

Comment:
If we have a colon here, we probably also want one for the public page url

Reply:
The public page url already has a colon from the localization string (at least in Portuguese), that's why I've added it to the second url

Reply:
Since we're setting the language to English with no way of changing it at the moment, we should go with how it's when using English, which is no colon by default. I'll then leave it up to you whether you want to add the colon or not.

Reply:
I've checked and it doesn't have a colon on the English one 🤡. Two different patterns

Reply:
Then let's just go with that, no colons